llvm/include/llvm/MC/MCSymbolRepo.h

```diff
@@ -23,6 +23,21 @@ class MCSymbolRepo : public MCSymbol {
 
   // A pointer to the TiketNode metadata of the corresponding GlobalObject.
   const TicketNode *CorrespondingTicketNode = nullptr;
+
+  // Get the symbol full name.
+  static const std::string getFullName(const MCContext &Ctx,
```

Comment:
Could we avoid converting the Twine to a std::string?

Reply:
As mentioned in [Twine](https://llvm.org/doxygen/classllvm_1_1Twine.html#details), “A Twine is not intended for use directly and should not be stored, its implementation relies on the ability to store pointers to temporary stack objects which may be deallocated at the end of a statement. Twines should only be used accepted as const references in arguments, when an API wishes to accept possibly-concatenated strings.” Therefore, we can't return a Twine.

Reply:
Good point. Fixed by using `std::string` with an initial `reserve()`.